src/web/modules/gateway_accounts/overview.njk

```diff
@@ -1,34 +1,34 @@
-{% extends "layout/layout.njk" %} 
+{% extends "layout/layout.njk" %}
 
 {% block main %}
+  <span class="govuk-caption-m">GOV.UK Pay platform</span>
   <h1 class="govuk-heading-m">Gateway accounts</h1>
 
   <table class="govuk-table">
     <thead class="govuk-table__head">
       <tr class="govuk-table__row">
-        <th class="govuk-table__header" scope="col">ID / External ID</th>
-        <th class="govuk-table__header" scope="col">Type</th>
-        <th class="govuk-table__header" scope="col">Service</th>
+        <th class="govuk-table__header" scope="col">ID</th>
         <th class="govuk-table__header" scope="col">Provider</th>
-        <th class="govuk-table__header" scope="col">Analytics ID</th>
+        <th class="govuk-table__header" scope="col">Service</th>
+        <th class="govuk-table__header" scope="col">Type</th>
       </tr>
     </thead>
     <tbody class="govuk-table__body">
       {% for account in accounts | sort(true, false, 'gateway_account_id') %}
       <tr class="govuk-table__row">
         <td class="govuk-table__cell">
-            <a class="govuk-link govuk-link--no-visited-state" href="/gateway_accounts/{% if account.gateway_account_external_id %}{{ account.gateway_account_external_id }}{% else %}{{ account.gateway_account_id }}{% endif %}">
-                {% if account.gateway_account_external_id %}{{ account.gateway_account_external_id }}{% else %}{{ account.gateway_account_id }}{% endif %}
-            </a>
+          {{ account.gateway_account_id }}
         </td>
-        <td class="govuk-table__cell">{{ account.type | capitalize }}</td>
-        <td class="govuk-table__cell">{{ account.service_name | truncate(30) }}</td>
         <td class="govuk-table__cell">{{ account.payment_provider | capitalize }}</td>
-        <td class="govuk-table__cell"><code>{{ account.analytics_id }}</code></td>
```

Comment:
Why is it removed?

Reply:
This is a high level list of all accounts, no one is using it I just put it there because it looked interesting. You can get the analytics ID for an account by going through to the details.